src/ViewportInputsStore.js

```diff
@@ -26,9 +19,12 @@ RootStore = types
         highlightedLink: 0, // we will compare linkColumns
         maximumHeightThisFrame: 150,
         cellToolTipContent: "",
-        jsonName: 'Athaliana_12_individuals_w100000',
-        startChunkURL: 'test_data/Athaliana_12_individuals_w100000/chunk00_bin100000.schematic.json',
-        endChunkURL: 'test_data/Athaliana_12_individuals_w100000/chunk01_bin100000.schematic.json'
+        jsonName: 'run1.B1phi1.i1.seqwish.w100',
+        startChunkURL: 'test_data/run1.B1phi1.i1.seqwish.w100/chunk0_bin100.schematic.json',
+        endChunkURL: 'test_data/run1.B1phi1.i1.seqwish.w100/chunk1_bin100.schematic.json',
+        pathNucPos: types.optional(PathNucPos, {path: "path", nucPos: 0}), // OR: types.maybe(PathNucPos)
+        pathIndexServerAddress: 'http://193.196.29.24:3010/',
```

Comment:
Server address.

Reply:
Is it possible to change the server address from the external source, e.g., environment variable?

Reply:
I am now wondering how to integrate with pipeline...

Reply:
As this code will be run within a browser, the browser itself can't read any environment variables, I am afraid.....

The only non-ui solution I see is that we have a `server.json` which has one property: `pathIndexServerAddress' listing the server.
Or we allow a listing of server addresses by given name. And the user can select via dropdown menu in the ui the server.

What would suite you best?

Reply:
Or we could make use of https://www.robertcooper.me/front-end-javascript-environment-variables#create-react-app.

Reply:
But a browser won't be able to read an environment variable set by the OS. Security restrictions.

Reply:
Hmmm, I use `sed` for modifying startChunkURL and endChunkURL at now pipeline while knowing that's not pretty good. It's also possible to modify pathIndexServerAddress by using `sed`.

Reply:
I saw u doing that. And I did not like it, either, but I was not aware of any better solution......
Any ideas @josiahseaman or @superjox ?

Reply:
If no one has a better idea, I would close this one tomorrow morning and merge into master.
This would mean we are only left with `sed` @6br

Reply:
Okay